Sources/StreamChat/Config/ChatClientConfig.swift

```diff
@@ -66,6 +66,21 @@ public struct ChatClientConfig {
     /// Is `true` by default.
     public var shouldConnectAutomatically = true
     
+    /// If set to `true`, the `ChatClient` will try to stay connected while app is backgrounded.
+    /// If set to `false`, websocket disconnects immediately when app is backgrounded.
```

Comment:
nit: Would be fine to mention that the client starts starts a background task for this